src/System.Management.Automation/engine/SessionStateProviderAPIs.cs

```diff
@@ -109,20 +67,6 @@ internal void AddSessionStateEntry(SessionStateProviderEntry providerEntry)
             );
         }
 
-        /// <summary>
-        /// Internal method used by RunspaceConfig for updating providers.
-        /// </summary>
-        /// <param name="providerConfig"></param>
-        private ProviderInfo AddProvider(ProviderConfigurationEntry providerConfig)
```

Comment:
It looks like RemoveProvider on line 1523 should be removed too, since it is only used for RunspaceConfiguration.

Reply:
Will remove

Reply:
based on our code coverage that code has never been hit